postgresql_user module: fix documentation

Andersson007 opened the pull request:
##### SUMMARY
Related with https://github.com/ansible/ansible/issues/47048

Fix postgresql_user module documentation to avoid ambiguity.
##### ISSUE TYPE
<!--- Pick one below and delete the rest -->
- Docs Pull Request

##### COMPONENT NAME
<!--- Write the short name of the module, plugin, task or feature below -->
postgresql_user

Andersson007 committed:
postgresql_user module: fix documentation

cmfrtl commented:
shipit

saxus commented:
shipit

Andersson007 commented:
bot_status

Andersson007 commented:
easyfix

cmfrtl reviewed (commented):
nice

Andersson007 commented:
shipit

acozine commented on lib/ansible/modules/database/postgresql/postgresql_user.py:
```diff
@@ -17,12 +17,14 @@
-     a PostgreSQL cluster. Privilege assignment, or removal, is an optional
+   - Adds or removes a user (role) from a remote PostgreSQL server instance
+     ("cluster" in PostgreSQL terminology) and, optionally,
+     grant the user access to an existing database or tables.
```
```suggestion
     grants the user access to an existing database or tables.
```
to match the tense of "adds or removes"

acozine reviewed (commented):
With the exception of the one verb-tense mistake / typo, this looks like an improvement. If you know of a good link for details on the difference between a Postgresql "role" and a Postgresql "user", a link to that would be helpful. But I'm okay merging this without that addition.

Andersson007 committed:
postgresql_user module: fix documentation - fix typo, add a link

Andersson007 commented:
@acozine 
> With the exception of the one verb-tense mistake

Fixed
> If you know of a good link for details on the difference between a Postgresql "role" and a Postgresql "user", a link to that would be helpful.

Added

Thank you for the review!

kustodian commented:
shipit

andytom commented:
Fixes #47048 

shipit
